packages/neuron-wallet/src/utils/lockUtils.ts

```diff
@@ -0,0 +1,96 @@
+import nodeService from '../startup/nodeService'
+import { CellOutPoint, OutPoint, Script } from '../appTypes/types'
+
+const { core } = nodeService
+
+export default class LockUtils {
+  static systemScriptInfo: { codeHash: string; outPoint: OutPoint } | undefined
+
+  static async systemScript() {
+    if (this.systemScriptInfo) {
+      return this.systemScriptInfo
+    }
+
+    const genesisHash: string = await core.rpc.getBlockHash('0')
+    const genesisBlock = await core.rpc.getBlock(genesisHash)
```

Comment:
We should use `getBlockByNumber('0')` to reduce one RPC call.

Reply:
updated, thanks